rs-e2e/src/main/java/gov/hhs/cdc/trustedintermediary/rse2e/AzureBlobFileFetcher.java

```diff
@@ -47,12 +47,25 @@ public static FileFetcher getInstance() {
         return INSTANCE;
     }
 
+    //  TODO - we need to modify the guts to only grab the golden or automated path
     @Override
     public List<HL7FileStream> fetchFiles() {
+        String files_path = System.getenv("LOCAL_FILE_PATH");
```

Comment:
Just a reminder to add instructions to the readme on how to set up the LOCAL_FILE_PATH env var. You could also consider adding it to `generate_env.sh`

Reply:
I didn't realize this changes are in AzureBlobFileFetcher. This class is meant only for blobs in Azure. To grab local files, the correct one is LocalFileFetcher